pinot-common/src/main/java/org/apache/pinot/common/utils/helix/PinotHelixPropertyStoreZnRecordProvider.java

```diff
@@ -51,6 +51,10 @@ public static PinotHelixPropertyStoreZnRecordProvider forSegments(ZkHelixPropert
     return new PinotHelixPropertyStoreZnRecordProvider(propertyStore, "/SEGMENTS");
   }
 
+  public static PinotHelixPropertyStoreZnRecordProvider forLogicalTable(ZkHelixPropertyStore<ZNRecord> propertyStore) {
```

Comment:
Where is this function used ? browser search failed me.

Reply:
This used in the `PinotHelixResourceManager#getAllLogicalTableNames`. previous it was parsing all the logical table configs, now it only reads the files and determines it. The same approach is used in the get schema as well.